lms/templates/commerce/receipt.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="wrapper-content-main payment-confirmation-step">
   <article class="content-main">
     <h3 class="title">
-      <%= interpolate(
+      <%- edx.StringUtils.interpolate(
             gettext( "Thank you %(full_name)s! We have received your payment for %(course_name)s."),
             { course_name: "<span class='course_name_placeholder'></span>",
               full_name: "<span class='full_name_placeholder'></span>"}, true
```

Comment:
You should be using HtmlUtils.interpolate() here. StringUtils would not protect your span tags and this code would cause span tag to be escaped. See this http://ui-toolkit.edx.org/utilities/html-utils/. Please if possible do test your change to be sure.